common/lib/xmodule/xmodule/xml_module.py

```diff
@@ -429,6 +429,12 @@ def add_xml_to_node(self, node):
         """
         # Get the definition
         xml_object = self.definition_to_xml(self.runtime.export_fs)
+
+        # If xml_object is None, we don't know how to serialize this node, but
+        # we shouldn't crash out the whole export for it.
+        if xml_object is None:
```

Comment:
Should we log a warning here as well?

Reply:
I think it's okay without. Right now, it'd be 1:1 with the existing warning.